packages/core/contracts/insured-bridge/RateModelStore.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../common/implementation/MultiCaller.sol";
+
+/**
+ * @title Maps rate model objects to L1 token.
+ * @dev This contract is designed to be queried by off-chain relayers that need to compute realized LP fee %'s before
+ * submitting relay transactions to a BridgePool contract. Therefore, this contract does not perform any validation on
+ * the shape of the rate model, which is stored as a string to enable arbitrary data encoding via a stringified JSON
+ * object. This leaves this contract unopionated on the parameters within the rate model, enabling governance to adjust
+ * the structure in the future.
+ */
+contract RateModelStore is Ownable, MultiCaller {
+    struct RateModel {
+        uint256 startBlock; // When the new rate model becomes active.
+        string oldRateModel; // Store old rate model so that public getter method does not need to modify state to
```

Comment:
is the convention for these strings always meant to be json parsable?

Reply:
Yeah, I think clients should be able to JSON.parse the string or error out